exercises/verify.cpp

```diff
@@ -11,8 +11,29 @@ using namespace std;
 
 int main()
 {
-  // placeholder
+  int num;
+  bool isPresent = false;
+  cout << "Insert number ";
+  cin >> num;
+
   int N[10] = {3, 4, 5, 1, 2, 3, 4, 9, 13, 0};
 
+  for (int i = 0; i < 10; i++)
+  {
+    if (num == N[i])
+    {
+      isPresent = true;
+      break;
+    }
+  }
+
+  if (isPresent)
+  {
+    cout << "The number " << num << " is present";
+  }
+  else
+  {
+    cout << "The number " << num << " is NOT present";
+  }
   return 0;
```

Comment:
Il return 0 è poco esplicativo di per sè.
Potresti fare un:
`#define EXIT_SUCCESS 0` e inserire alla fine del main `return EXIT_SUCCESS;` in modo tale da incrementare la leggibilità del codice.